plugin.video.retrospect/channels/plugin.video.retrospect.channel.de/srf/chn_srf.json

```diff
@@ -0,0 +1,16 @@
+{
+  "channels": [
+    {
+      "guid": "A9BE95B0-2ED6-4E75-B5DF-C3ECAF0441E6",
+      "name": "SRF.ch Play",
+      "description": "Schweizer Radio und Fernsehen Play content.",
```

Comment:
This would require a rather extensive change. As the current .json solution was introduced to make it all load a lot faster when listing the main channels. I will have to see how the localization would work, but I have ideas for that.

Reply:
keep it in mind for a future update, not mandatory at this stage